etc/kayobe/containers/pulp/post.yml

```diff
@@ -4,8 +4,8 @@
     url: "{{ pulp_url }}/pulp/api/v3/status/"
   register: pulp_status
   until: pulp_status is success
-  retries: 30
-  delay: 2
+  retries: "{{ pulp_timeout_retries | default(30) }}"
+  delay: "{{ pulp_timeout_delay | default(2) }}"
```

Comment:
Let's increase the default enough to fix your problem @Alex-Welsh

Reply:
Done